README.md

```diff
@@ -1,5 +1,7 @@
 # Espresso Interface
 
+[![Build Status](https://travis-ci.org/Dhii/espresso-interface.svg?branch=master)](https://travis-ci.org/Dhii/espresso-interface)
+
```

Comment:
Didn't yet add the repo to CodeClimate. Currently checking if we can share administration of repos registered in CC.